code/game/mecha/equipment/tools/tools.dm

```diff
@@ -1,10 +1,12 @@
+#define THE_NUMBER_WHICH_THE_SLEEP_TIME_OF_DO_AFTER_COOLDOWN_IS_TO_BE_DIVIDED_BY_WHEN_USING_AN_EXOSUIT_MOUNTED_DRILL_TO_DIG_SAND 2

```

Comment:
Do you think this is some kind of joke
Do you think I talk about coding standards because I'm having a laugh

The point of my name is that it's both ironic and not ironic at the same time